src/store.js

```diff
@@ -63,6 +96,21 @@ export default new Vuex.Store({
     },
     getNextProductId: state => {
       return state.nextProductId;
+    },
+    getCartProductsCount: state => {
+      return state.cart.count;
+    },
+    getCartProducts: state => {
+      let products = []
+
+      state.cart.products.forEach((product) => {
+        products.push({
+          data: state.allProducts.find(item => item.id == product.productId),
```

Comment:
Dodawaj do koszyka producty od razu z data (zakładam, że to opis, obrazek itp), albo zrób osobny getter który zwraca `data` dla podanego ID